_includes/navbar.html

```diff
@@ -20,6 +20,7 @@
         <li><a href="/documentation.html">Users Guide</a></li>
         <li><a href="/qutip-tutorials/">Tutorials</a></li>
         <li><a href="/qutip-benchmark/">Benchmarks</a></li>
+        <li><a href="/qutip-virtual-lab.html">QuTiP Virtual Lab</a></li>
```

Comment:
I think the `.html` will break this link
```suggestion
        <li><a href="/qutip-virtual-lab">QuTiP Virtual Lab</a></li>
```

Reply:
This is a link to the HTML file with the blurb.